source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/RenderedAdaptiveCard.java

```diff
@@ -53,6 +53,22 @@ public void registerInputHandler(IInputHandler handler)
         return input;
     }
 
+    public void setInputs(Map <String, String> inputs)
```

Comment:
[nit] extra space between "Map" and "<key, val>"

Reply:
Resolved